splinterd/sample_configs/splinterd.toml.example

```diff
@@ -128,3 +128,32 @@ oauth_client_secret = "<client-secret>"
 
 # Redirect URL for the OAuth provider used by the REST API
 oauth_redirect_url = "http://localhost:8080/oauth/callback"
+
+
+#Appenders is the structure and stdout is the items name
+[appenders.stdout]
+#Type of appender options are stdout,stderr,file,rolling_file
+kind = "stdout"
+#Pattern controls the formatting of each log message.
+#Full reference `https://docs.rs/log4rs/1.0.0/log4rs/encode/pattern/index.html`
+pattern = "[ {d(%Y-%m-%d %H:%M:%S%.3f)}] T[{T}] {l} [{M}] {m}\n"
+
+[appenders.stderr]
+kind = "stderr"
+
+[appenders.rolling_file]
+kind = "rolling_file"
+#Filename is specific to rolling_file and file appenders and is simply a path to a file.
+filename = "/var/logs/my_file.log"
+#Size is specific to rolling_file and specifies when the file should be rolled/overwritten.
+size = "16.0M"
+
+#loggers is the structure and splinter is the items name
```

Comment:
This comment doesn't really explain anything since a sysadmin probably isn't going to know the terminology used here.